site/visitor-console/src/pages/SignInSuccess.tsx

```diff
@@ -1,14 +1,17 @@
 import Countdown from "react-countdown";
 import { Link, useNavigate, useSearchParams } from "react-router-dom";
 import PageCard from "../components/PageCard";
+import QuizProgress from "../components/QuizProgress";
 
 const SignInSuccess = () => {
   const navigate = useNavigate();
   let [searchParams] = useSearchParams();
   const next_page = (searchParams.get("next") as string) || "/";
 
   const renderer = () => (
-    <PageCard title="Sign-In Sucessful">
+    <PageCard title="Sign-In Successful">
+      <h1>Your Safety Quiz Progress</h1>
+      <QuizProgress />
```

Comment:
This automatically times out and goes back to the login page after 10 seconds?

Reply:
Yes, after a user signs in, it goes back to the login page so another user can sign in after them.